src/rust/engine/Cargo.lock

```diff
@@ -2936,11 +3123,34 @@ name = "time"
 version = "0.1.42"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 dependencies = [
- "libc 0.2.66 (registry+https://github.com/rust-lang/crates.io-index)",
+ "libc 0.2.68 (registry+https://github.com/rust-lang/crates.io-index)",
  "redox_syscall 0.1.56 (registry+https://github.com/rust-lang/crates.io-index)",
  "winapi 0.3.8 (registry+https://github.com/rust-lang/crates.io-index)",
 ]
 
+[[package]]
```

Comment:
It looks like something went awry in the lockfile: this change should not have added back `tokio 0.1`.

Most likely you'll want to revert this to master's lockfile, run a build, and then commit only the edit that that causes. If that _still_ causes `tokio 0.1` to be pulled in, then one of the deps in `Cargo.toml` might be incorrect.